open_src/influx/coordinator/statement_executor.go

```diff
@@ -2216,12 +2222,26 @@ func (e *StatementExecutor) executeDropStream(stmt *influxql.DropStreamsStatemen
 	return e.MetaClient.DropStream(stmt.Name)
 }
 
-func (e *StatementExecutor) executeShowConfigs(stmt *influxql.ShowConfigsStatement) error {
-	return fmt.Errorf("impl me")
+func (e *StatementExecutor) executeShowConfigs(stmt *influxql.ShowConfigsStatement) (models.Rows, error) {
+	row := &models.Row{Columns: []string{"component", "instance", "name", "value"}}
+	row.Values = append(row.Values, []interface{}{sqlConfig, app.BindAddress, loggingLevel, logger.Alevel})
```

Comment:
Do not add global variable `app.BindAddress`.

Suggest to add `hostname` to StatementExecutor structure.

The hostname should like:
`hostname := config.CombineDomain(s.config.HTTP.Domain, s.config.HTTP.BindAddress)`